db/seeds.rb

```diff
@@ -5,11 +5,85 @@
   User.invite!(email: email) unless User.find_by(email: email)
 end
 
-5.times do
-  client = FactoryBot.create :client
-  4.times do
-    identity = FactoryBot.create(:identity, client: client)
-    FactoryBot.create_list :departure, 10, identity: identity
-    FactoryBot.create_list :arrival, 10, identity: identity
-  end
-end
+
+# 5.times do
+#   client = FactoryBot.create :client
+#   4.times do
+#     identity = FactoryBot.create(:identity, client: client)
+#     FactoryBot.create_list :departure, 10, identity: identity
+#     FactoryBot.create_list :arrival, 10, identity: identity
+#   end
+# end
+
+###########################
+
+# Male
+# Lee Jun-fan
+# Dual Citizenship
+
+# Identity 1
+# Born: California
+# Nationality: United States
+
+# Identity 1
+# Born: California
+# Nationality: Hong Kong
+
+
+
+#### Lee Jun-fan ####
+
+Country.create(id: 1, name: "United States of America")
+Client.create(id: 1, im_client_id: "12345", file_number: "1")
+Identity.create(id: 1, client_id: 1, identity_number: "Y-3183346-E", family_name: "Lee", first_name: "Bruce", second_name: "", third_name: "", gender: "Male", country_of_birth_id: 1, nationality: "United State", issuing_state_id: 1, serial_number: "Y-3775136-E")
+
+Country.create(id: 2, name: "Hong Kong")
+Identity.create(id: 2, client_id: 1, identity_number: "Y-3183347-E", family_name: "Lee", first_name: "Jun-fan", second_name: "", third_name: "", gender: "Male", country_of_birth_id: 1, nationality: "Hong Kong", issuing_state_id: 2, serial_number: "Y-3775137-E")
+
+# Identity 1
+Movement.create(identity_id: 1, direction: "departure", carrier_date_time: "2007-09-26 20:00:00")
+Movement.create(identity_id: 1, direction: "arrival", carrier_date_time: "2007-10-18 06:00:00")
```

Comment:
you could do `carrier_date_time: 100.days.ago`
and then on the next line:
:arrival `carrier_date_time: 88.days.ago`

and then next
`carrier_date_time: 100.days.ago`